code/modules/mob/living/carbon/carbon.dm

```diff
@@ -303,8 +303,8 @@
 				H.update_inv_gloves(0)
 			H.gloves.germ_level = 0
 		else
-			if(H.bloody_hands)

```

Comment:
I did test using the old statement, did not work, I believe it is related to var change on bloody_hands.